src/main/java/frc/robot/subsystems/conveyor/commands/LoadConveyor.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+/**
+ * This command load the conveyor, as long as the conveyor isn't "out of space".
+ */
+public class LoadConveyor extends CommandBase {
+    private final Conveyor conveyor;
+    private final double power;
+
+    public LoadConveyor(Conveyor conveyor, double power) {
```

Comment:
Notice that you want to move the conveyor the minimal distance to enter each power cell in order to optimize our space usage